requirements/constraints.txt

```diff
@@ -9,11 +9,13 @@
 # linking to it here is good.
 
 # use edx-platform package versions.
+celery>=5.2.2,<=5.2.7
```

Comment:
@alexjmpb Why isn't this requirement fixed to the 5.2.7 version being used on edx-platform?